blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/Configuration.java

```diff
@@ -65,6 +65,20 @@ public void init() throws IOException {
         return Collections.unmodifiableMap(props);
     }
 
+    public Map<Object, Object> getAllProperties() {
```

Comment:
`loadDefaults` gets called by the various `enum` configuration types during static initialization. However, once that's already happened, the map returned from `getProperties` will not have any of those default values in it.

For what it's worth, the `getProperties` and `getAllProperties` methods are only ever called from `ConfigurationTest`. They are a kind of diagnostic, necessary to tell what's going on inside the `Configuration` class, based on how its internals work now. Since I intend to fix the internals of the class, and make them far more sane, they shouldn't be necessary in the future. I expect them to be removed at some point during the course of refactoring.